source/dotnet/Library/AdaptiveCards.Templating.Net6/AdaptiveCards.Templating.Net6.csproj

```diff
@@ -44,8 +44,8 @@
     <None Remove="..\AdaptiveCards.Templating\AdaptiveCardsTemplatingLexer.tokens" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AdaptiveExpressions" Version="4.11.2" />
-    <PackageReference Include="Antlr4.Runtime.Standard" Version="4.8.0" />
+    <PackageReference Include="AdaptiveExpressions" Version="4.20.1" />
```

Comment:
Can you check that the version is not overripened in the nuspec? 
https://github.com/microsoft/AdaptiveCards/blob/384f3d1064382021f10303445f70680ad07a49d2/source/dotnet/NuGet/AdaptiveCards.Templating.nuspec#L15